File: common/src/main/java/dev/vini2003/hammer/config/HC.java
package dev.vini2003.hammer.config;

import net.fabricmc.api.ModInitializer;

public class HC {
	public static void onInitialize() {
		dev.vini2003.hammer.core.HC.LOGGER.info("Initialized '" + dev.vini2003.hammer.core.HC.CONFIG_MODULE_ID + "' module.");
	}
}

File: hammer-chat/src/main/kotlin/dev/vini2003/hammer/chat/HC.kt
package dev.vini2003.hammer.chat

import dev.vini2003.hammer.chat.registry.common.HCCommands
import dev.vini2003.hammer.chat.registry.common.HCEvents
import dev.vini2003.hammer.chat.registry.common.HCNetworking
import net.fabricmc.api.ModInitializer

object HC : ModInitializer {
	override fun onInitialize() {
		HCNetworking.init()
		HCCommands.init()
		HCEvents.init()
	}
	
}